app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_post_comments_counter
+    post.comments_counter = post.comments.count
+    post.save
+  end
```

Comment:
- [ ] [optional] @Salwa99 great work on your update_post_likes_counter method, however, I would recommend you t use [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_post_likes_counter method every time a new post has been saved to the database, then make the update_post_likes_counter method private